research-software-directory/RSD-as-a-service
# Fix some SonarCloud issues

## Fix some SonarCloud issues

### Changes proposed in this pull request

* Fix some low-effort SonarCloud issues
* Add a logger to the authentication module

### How to test

* `docker compose down --volumes && docker compose build --parallel && docker compose up --scale data-generation=1`
* The build should succeed without errors
* Sign in as admin and user, through various providers, this should still work

PR Checklist:
